.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I think this got accidentally checked in and should be removed (and gitignored).

If there are plans for us to use shared settings, we should add this file when it has such settings.

Reply:
Removed and added to .gitignore

Reply:
This file is still showing up as of 6b1f8706beab9bfbc949ac6b144b0f21f47d378b.